### DISCUSSION TITLE
Provide more documentation around container security

### DISCUSSION THREAD

**Discussion by levlaz** [dagger/dagger#12467] [Ideas]
### What is the issue?

Our docs do a good job discussing why its _necessary_ to have a privileged container (https://github.com/dagger/dagger/blob/main/core/docs/d7yxc-operator_manual.md) but we do not mention why it is _safe_ to do so. 

We should provider some additional documentation so that  security teams can have more context to make the right infrastructure decisions for their organization.

**Comment by iainlbc** [comment]
Hi @levlaz , we will be undergoing security review soon of our proposed solution based on dagger. It would be useful to have any additional info around this topic if possible.
